engine/crates/telemetry/src/graceful_shutdown.rs

```diff
@@ -0,0 +1,4 @@
+pub fn graceful_shutdown() {
+    opentelemetry::global::shutdown_tracer_provider();
```

Comment:
would need to shut down the meter provider but that's also only available through the "SDK" implementation, not the interface -_-"

Reply:
can't even shutdown simultaneously lovely.

Reply:
Yes it's challenging to wire that into this function, but I could spend some more time and try harder.

Reply:
Honestly you can probably leave it as is, I'll remove the reload logic, so can probably keep the SdkMeter ref somewhere